test/com/bjorn/MakeChange/test/MakeChangeTest.java

```diff
@@ -0,0 +1,111 @@
+package com.bjorn.MakeChange.test;
+
+import com.bjorn.MakeChange.MakeChange;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.HashMap;
+
+import static org.hamcrest.CoreMatchers.instanceOf;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by bjornjohnson on 1/5/16.
+ */
+public class MakeChangeTest {
+
+    private MakeChange makeChange;
+    private HashMap<String, Integer> coinAmountHashMap;
+
+    @Before
+    public void setUp() {
+        makeChange = new MakeChange();
+        coinAmountHashMap = new HashMap<String, Integer>();
+    }
+
+    @Test
+    public void testInstanceOf() {
+        assertThat(makeChange, instanceOf(MakeChange.class));
+    }
+
+    @Test
+    public void testZeroCurrency() {
+        assertEquals(coinAmountHashMap, makeChange.makeChange(0));
+    }
+
+    @Test
+    public void testOneCent() {
+        coinAmountHashMap.put("P", 1);
```

Comment:
Since one of the parameters to `makeChange` is also called `coinAmountHashMap`, this name is a little confusing, especially if you're bouncing back and forth between test and production code. Try to avoid shadowing existing names in a different context. In tests, I often use "expected" to clarify which variables hold the expected values, e.g.:

```java
@Test
public void testOneCent() {
  expectedChangeAmounts.put("P", 1);
  assertEquals(expectedChangeAmounts, makeChange.makeChange(1));
}
```